style.css

```diff
@@ -531,3 +579,640 @@ h4 {
 .social img {
   padding-right: 33px;
 }
+
+/* MEDIA FOR MOBILE */
+
+@media (max-width: 768px) {
+  .menu {
+    display: none;
+  }
+
+  .image-big {
+    display: none;
+  }
+
+  .image-dot {
+    display: none;
+  }
+
+  .component-1 {
+    display: none;
+  }
+
+  .component2 {
+    display: none;
+  }
+
+  .keywords-first {
+    display: none;
+  }
+
+  .keywords-second {
+    display: none;
+  }
+
+  .buttonbox-first {
+    display: none;
+  }
+
+  .buttonbox-second {
+    display: none;
+  }
+
+  .geometry-center {
+    display: none;
+  }
+
+  .pc-left {
+    display: none;
+  }
+
+  .pc-left-bigger {
+    display: none;
+  }
+
+  .pc-right-squares {
+    display: none;
+  }
+
+  .pc-left-squares {
+    display: none;
+  }
+
+  .dot-right-yellow {
+    display: none;
+  }
+
+  .pc-right-yellow {
+    display: none;
+  }
+
+  .dots-teal {
+    display: none;
+  }
+
+  .logo-form-desk {
+    display: none;
+  }
+
+  .p-desktop {
+    display: none;
+  }
+
+  .img-form {
+    display: none;
+  }
+
+  .dot-projects {
+    display: none;
+  }
+
+  .bckrg-form {
+    display: none;
+  }
```

Comment:
- [ ] You should use [DRY principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) here. You assign the `display: none;` property to a class and assign that class to all the elements rather than repeatedly assigning the property.